src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,169 @@
+// React Imports
+import React from 'react';
+import { Link } from 'react-router-dom';
+// Solid Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import AppBar from '@mui/material/AppBar';
+import Badge from '@mui/material/Badge';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+// Custom Component Imports
+import NavMenu from './NavMenu';
+
+/**
+ * NavBar Component - Component that generates NavBar section for PASS
+ *
+ * @memberof GlobalComponents
+ * @name NavBar
+ */
+
+const NavBar = () => {
+  const { session } = useSession();
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
Use `useState` instead of `React.useState` for consistency with other files. So import React, { useState } from "react' on Line 2.

This is true for Lines 30, 31, and 32.